code-gen-projects/java/code-gen-demo/src/test/java/org/example/CodeGenTest.java

```diff
@@ -137,4 +176,76 @@ class CodeGenTest {
             }
         }
     }
+
+    @Test void roundtripGoodTestForSequence() throws IOException {
+        File dir = new File(System.getenv("ION_INPUT") + "/good/sequence");
+        String[] fileNames = dir.list();
+        for (String fileName : fileNames) {
+            File f = new File(dir, fileName);
+            InputStream inputStream = new FileInputStream(f);
```

Comment:
None of the test files are very large, but you might consider wrapping the `FileInputStream` with a `BufferedInputStream` to avoid future slowdowns.